app/doc/responses.go

```diff
@@ -87,6 +87,20 @@ type CreatedResponse struct {
 	Success bool `json:"success"`
 }
 
+// The request has succeeded. The message shows if the object has been deleted.
+// swagger:response createdOrNotChangedResponse
+type createdOrNotChangedResponse struct {
+	// in: body
+	Body struct {
+		// enum: created,not changed
+		// required: true
+		Message string `json:"message"`
```

Comment:
Same remark as I did yesterday about the boolean. I have to retrieve the PR to see if you have answered... 😛

Reply:
I don't remember there was a comment about the boolean (don't see a boolean here)...

Reply:
I mentioned somewhere that it might be useful (for the frontend typically) to have a boolean in the "data" map to know if it is "created" of "not changed", so that the client app does not have to string comparison on the message.